akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -211,20 +211,41 @@ akka.cluster.auto-down-unreachable-after = 120s
 This means that the cluster leader member will change the `unreachable` node
 status to `down` automatically after the configured time of unreachability.
 
-This is a naïve approach to remove unreachable nodes from the cluster membership. It
-works great for crashes and short transient network partitions, but not for long network
-partitions. Both sides of the network partition will see the other side as unreachable
-and after a while remove it from its cluster membership. Since this happens on both
-sides the result is that two separate disconnected clusters have been created. This
-can also happen because of long GC pauses or system overload.
+This is a naïve approach to remove unreachable nodes from the cluster membership.
+It can be useful during development but in a production environment it causes
```

Comment:
it causes => it can (and often will) cause

Reply:
My concern with "it can" is that it suggests that in some cases it may be okay to run with auto-downing in production. I think, given the consequences of doing that (whether it's corruption, or just cluster breakdown), it would be preferable to make it clear that running with auto-downing will cause problems at some point.